Gemfile

```diff
@@ -42,8 +42,8 @@ end
 
 
 gem 'autoprefixer-rails'
-gem 'bootstrap-sass', '~> 3.3'
-gem 'bootstrap_form', '~> 2.5'
+gem 'bootstrap'
+gem 'bootstrap_form'
```

Comment:
Add back version qualifier but to latest required i.e. `~> 2.7` if that is the required version...